internal/blocksync/pool.go

```diff
@@ -630,6 +629,10 @@ func (bpr *bpRequester) redo(peerID types.NodeID) {
 // Responsible for making more requests as necessary
 // Returns only when a block is found (e.g. AddBlock() is called)
 func (bpr *bpRequester) requestRoutine(ctx context.Context) {
+	timer := time.NewTimer(0)
+	<-timer.C
```

Comment:
I don't believe this is necessary because the timer is always reset before it's used.

Reply:
As above.